guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -308,7 +308,7 @@ This is the only method with which you can provision hosts in IPv6 network.
 .Prerequisites
 
 * Ensure that you meet the requirements for HTTP booting.
-For more information, see {BaseURL}/planning_for_red_hat_satellite/chap-red_hat_satellite-architecture_guide-provisioning_concepts#http_booting_requirements[HTTP Booting Requirements] in _Planning for {Project}_.
+For more information, see {PlanningDocURL}http-booting-requirements[HTTP Booting Requirements] in _Planning for {Project}_.
```

Comment:
I added a new ID for this section in `guides/doc-Planning_Guide/topics/Provisioning_Concepts.adoc`, this link will work after this PR is merged.